osu.Game/Overlays/OnScreenDisplay.cs

```diff
@@ -101,7 +101,7 @@ public void StopTracking(object source, ITrackableConfigManager configManager)
             DisplayTemporarily(box);
         });
 
-        private void displayTrackedSettingChange(SettingDescription description) => Display(new TrackedSettingToast(description));
+        private void displayTrackedSettingChange(SettingDescription description) => Scheduler.AddOnce(Display, new TrackedSettingToast(description));
```

Comment:
Is it going to be an issue that `AddOnce` only keeps the earliest arg?

Reply:
Maybe a very rare case. I think we should fix that framework side though (to always use the most recent) because it doesn't make sense as-is.

Reply:
I pondered this myself but then decided that if two tracked settings are going to get changed in a single frame then there's really no good/sane thing to do anyhow.

Reply:
I think this also applies to the same setting getting changed multiple times, ie via input events, which can potentially happen as a more likely scenario (where the last value is what we want):

https://github.com/ppy/osu/blob/8672b3325a776726a8533d6b134aae83182ef5d3/osu.Game.Rulesets.Mania/Configuration/ManiaRulesetConfigManager.cs#L32

Reply:
WIll be resolved via https://github.com/ppy/osu-framework/pull/4936